README.md

```diff
@@ -1,22 +1,22 @@
 <p align="center"><img src="https://gis4wrf.github.io/assets/images/gis4wrf_logo.png"></p>
 
-# GIS4WRF [![Build Status](https://travis-ci.com/GIS4WRF/gis4wrf.svg?branch=master)](https://travis-ci.com/GIS4WRF/gis4wrf)
+# GIS4WRF
 
-**Download a free copy of the GIS4WRF article at https://authors.elsevier.com/a/1YFYV4sKhEHMOB**
+| Build and Tests | Documentation | Article DOI | Software DOI |
+|-----------------|---------------|-------------|--------------|
+[![Build Status](https://travis-ci.com/GIS4WRF/gis4wrf.svg?branch=master)](https://travis-ci.com/GIS4WRF/gis4wrf) | [![docs](https://img.shields.io/badge/docs-gis4wrf.github.io-blue.svg)](https://gis4wrf.github.io) | [![DOI](https://img.shields.io/badge/DOI-10.1016%2Fj.envsoft.2018.10.018-brightgreen.svg)](https://doi.org/10.1016/j.envsoft.2018.10.018) | [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1288569.svg)](https://doi.org/10.5281/zenodo.1288569) |
 
-***For information on how to install GIS4WRF, or check out the main documentation and tutorials, please refer to the GIS4WRF website at https://gis4wrf.github.io.***
-
-## What is GIS4WRF?
-GIS4WRF is a free and open source [QGIS](https://qgis.org/) plug-in to help researchers and practitioners with their [Advanced Research Weather Research and Forecasting](https://www.mmm.ucar.edu/weather-research-and-forecasting-model) modelling workflows. GIS4WRF can be used to pre-process input data, run simulations, and visualize or post-process results. We offer MPI-enabled pre-built binary distributions for Windows, macOS and Linux through [WRF-CMake](https://github.com/WRF-CMake/WRF).
+GIS4WRF is a free and open source [QGIS](https://qgis.org/) plug-in to help researchers and practitioners with their [Advanced Research Weather Research and Forecasting](https://www.mmm.ucar.edu/weather-research-and-forecasting-model) modelling workflows. GIS4WRF can be used to pre-process input data, run simulations, and visualize or post-process results. We offer MPI-enabled pre-built binary distributions for Windows, macOS and Linux through [WRF-CMake](https://github.com/WRF-CMake/WRF). For information on how to install GIS4WRF, or check out the main documentation and tutorials, please refer to the GIS4WRF website at https://gis4wrf.github.io.
 
 <p align="center"><img src="https://gis4wrf.github.io/assets/images/gis4wrf-demo.gif"></p>
 <p align="center">Configure, run and visualize a WRF simulation in GIS4WRF</p>
 
-## How to reference GIS4WRF
-When using GIS4WRF, we kindly ask to please acknowledge our work by referencing both the article and the software.
+
+## Citing
+If you use GIS4WRF in a published work, please cite both the paper (https://doi.org/10.1016/j.envsoft.2018.10.018), and the specific version of GIS4WRF you are using (https://doi.org/10.5281/zenodo.1288569).
 
 ### Example
-Adapt using your referencing style from the example below
+
 ```
 Meyer, D., & Riechert, M. (2019). Open source QGIS toolkit for the Advanced Research WRF
```

Comment:
There's a type in both lines, a redundant comma before the &. Or is that how it's done?

Reply:
Thanks for checking but this is correct -- I followed APA here as it's now well accepted across fields (see https://publications.agu.org/agu-grammar-and-style-guide/#reference).